frontend/src/store.js

```diff
@@ -70,6 +70,11 @@ export default () => {
       statistics: [],
       bets: [],
       maps: [],
+      mapInfo: {
```

Comment:
spricht was dagegen, maps und mapInfo zu vereinen? kann das nicht alles zusammen in dem maps array stehen? sind da nicht auch map objekte drin, die auch mapDescr und civInfo enthalten koennen? sonst muss man immer zwischen maps und mapInfo mit id abfragen hin und her mappen.

Reply:
:heavy_check_mark: